spec/acceptance/peadm_spec/plans/test_migration.pp

```diff
@@ -30,6 +32,8 @@
     new_primary_host => $new_primary_target,
     upgrade_version => $upgrade_version,
     replica_host => $new_replica_target,
+    primary_postgresql_host => $primary_postgresql_target,
+    new_primary_postgresql_host => $new_primary_postgresql_target,
```

Comment:
the parameter in migrate.pp is called 'primary_postgresql_host' and we should pass 'new_primary_postgresql_target' into it, i.e. `primary_postgresql_host => $new_primary_postgresql_target,`